modules/crowdfundings/graphql/resolvers/_mutations/mergeUsers.js

```diff
@@ -137,19 +136,13 @@ module.exports = async (_, args, context) => {
 
     // remove old user
     await transaction.public.users.deleteOne({id: sourceUser.id})
-
     await transaction.transactionCommit()
 
     try {
-      unsubscribeFromMailchimp({
-        email: sourceUser.email
-      })
-      updateUserOnMailchimp({
-        userId: targetUserId,
-        pgdb
-      })
+      await unsubscribeEmail({ email: sourceUser.email })
+      await enforceSubscriptions({ pgdb, userId: targetUserId })
```

Comment:
the new user's email needs to be subscribed to the same newsletters, the old user had. I think this here is not enough. If the old hat DAILY and the new did not have it yet, it doesn't get it, right?

Reply:
hmmm, and what if the old had no newsletters (because manually changed) and the new one had some?